PIP_COMPATIBILITY.md

```diff
@@ -364,8 +364,8 @@ does support a large subset.
 Missing options and subcommands are prioritized based on user demand and the complexity of the
 implementation, and tend to be tracked in individual issues. For example:
 
-- [`--trusted-host`](https://github.com/astral-sh/uv/issues/1339)
-- [`--user`](https://github.com/astral-sh/uv/issues/2077)
+-   [`--trusted-host`](https://github.com/astral-sh/uv/issues/1339)
+-   [`--user`](https://github.com/astral-sh/uv/issues/2077)
```

Comment:
I cannot comprehend why this is changed.

Reply:
It renders fine though.

Reply:
Tragic:

- https://github.com/prettier/prettier/pull/15526
- https://github.com/prettier/prettier/issues/5019